webapp/src/app.tsx

```diff
@@ -1532,6 +1542,26 @@ export class ProjectView
         }
     }
 
+    onScaleChanged(oldScale: number, newScale: number) {
```

Comment:
passing in the oldScale here to set the initial scale seems kinda fragile. is the initial scale not always the same value? and failing that, can't we just read it from blockly when we load the workspace?

Reply:
It's different between Monaco and Blockly. I'll see about reading it when we load the workspace. Should be do-able.